Apps/Admin/Server/Services/DelegationService.cs

```diff
@@ -215,8 +181,9 @@ private async Task UpdateDelegationAsync(Dependent dependent, IList<string> dele
             }
 
             // Compare passed in delegate hdids with dependent allowed delegations in database to determine what allowed delegations to add.
+            IEnumerable<string> delegateHdidEnumerable = delegateHdids.ToList();
```

Comment:
I'd suggest moving this above line 174, and updating the reference on line 176.  I'd also suggest naming it `delegateHdidList` and making it of type `IList<string>`.

Reply:
Changed.